tests/Behat/SecurityContext.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Tests\Behat;
+
+use App\Entity\Organization;
+use App\Repository\OrganizationRepository;
+use App\Repository\UserRepository;
+use Behat\Behat\Context\Context;
+use Behat\Behat\Context\Environment\InitializedContextEnvironment;
+use Behat\Behat\Hook\Scope\BeforeScenarioScope;
+use Behat\Mink\Driver\BrowserKitDriver;
+use Behat\MinkExtension\Context\MinkContext;
+use Symfony\Bridge\Doctrine\Security\User\UserLoaderInterface;
+use Symfony\Component\BrowserKit\Cookie;
+use Symfony\Component\HttpFoundation\Session\SessionInterface;
+use Symfony\Component\Security\Core\Authentication\Token\UsernamePasswordToken;
+use Symfony\Component\Security\Core\Exception\UsernameNotFoundException;
+
+final class SecurityContext implements Context
+{
+    private UserRepository $userRepository;
+    private OrganizationRepository $organizationRepository;
+    private SessionInterface $session;
+    private MinkContext $minkContext;
+
+    public function __construct(UserRepository $userRepository, OrganizationRepository $organizationRepository, SessionInterface $session)
+    {
+        $this->userRepository = $userRepository;
+        $this->organizationRepository = $organizationRepository;
+        $this->session = $session;
+    }
+
+    /**
+     * @BeforeScenario
+     */
+    public function gatherContext(BeforeScenarioScope $scope): void
+    {
+        /** @var InitializedContextEnvironment $environment */
+        $environment = $scope->getEnvironment();
+        /** @var MinkContext $minkContext */
+        $minkContext = $environment->getContext(MinkContext::class);
+        $this->minkContext = $minkContext;
+    }
+
+    /**
+     * @Given I am authenticated as a user
```

Comment:
maybe you can allow to choose the user : `Given I am a user authenticated as "\w+"`, and `Given I am an organization authenticated as "\w+"`,

Reply:
`Given I am a user authenticated as "user1"`, `Given I am an organization authenticated as "UL 05"`

Reply:
Already done on line 64:
```
I am authenticated as "user1@resop.com"
I am authenticated as "UL 05"
```